grammars/dart.json

```diff
@@ -399,6 +399,7 @@
 		"string-interp": {
 			"patterns": [
 				{
+					"name": "string.interpolated.expression.dart",
```

Comment:
I just matched the name of the expression interpolation, as I wasn't sure if this should have its own name or not. Let me know your thoughts, thanks!